dip-0008.md

```diff
@@ -125,112 +129,125 @@ are InstantSend locked in a retroactive way if they were unknown before appearin
-succeeds or fails without performing additional attempts.
+### Signing blocks using single LLMQ
 
-The request id is `SHA256("clsig", blockHeight)` and the message hash is the block hash of the
-previously successful attempt.
+This is the default method which can be used by all nodes.
 
-After a LLMQ member has successfully recovered the final ChainLocks
-signature, it must create a P2P message and propagate it to all nodes. The
-message is called `CLSIG` and has the following structure:
+The request id in this case is `SHA256("clsig", blockHeight)`.
 
-| Field | Type | Size | Description |
-|--|--|--|--|
-| height | int32 | 4 | Height of the signed block |
-| blockHash | uint256 | 32 | Hash of the signed block from the successful attempt |
-| sig | BLSSig | 96 | Recovered signature |
+After a LLMQ member has successfully recovered the final ChainLocks
+signature, it must create a `CLSIG` P2P message and propagate it to all nodes
+through the inventory system.
 
-This message is propagated through the inventory system.
+### Signing blocks using multiple LLMQs
 
-Upon receipt, each node must perform the following verification before
-announcing it to other nodes:
+This is an optional method which can be used only by nodes running on protocol
+70220 or newer.
 
-  1. Based on the deterministic masternode list at the given height, a quorum must be
-  selected that was active at the time this block was mined
-  2. The signature must verify against the quorum public key and
-  `SHA256(llmqType, quorumHash, SHA256(height), blockHash)`. `llmqType` and `quorumHash`
-  must be taken from the quorum selected in 1.
+The request id for each LLMQ is calculated as `SHA256("clsig", blockHeight, quorumHash)`.
 
+After a LLMQ member has successfully recovered the final ChainLocks signature
+for its LLMQ, it must create a `CLSIG` P2P message and propagate it to non-SPV nodes
+through the inventory system.
 
 ## Handling of signed blocks
 
-When a new block has been successfully signed by a LLMQ and the `CLSIG` message
-is received by a node, it should ensure that only this block is locally
-accepted as the next block.
-
-If an alternative block for the same height is received, it must be invalidated
-and removed from the currently active chain since a signed block has already
-been received. If the correct block is already present locally, its chain
-should be activated as the new active chain. If the correct block is not known
-locally, it must wait for this block to arrive and request it from other nodes
-if necessary.
+The node should ensure that the fully signed block is locally accepted as the
+next block. If the correct block is already present locally, its chain should
+be activated as the new active chain tip. If the correct block is not known
+locally, the node must wait for this block to arrive and request it from other
+nodes if necessary. If an alternative block for the same height is received,
```

Comment:
Suggested rewording of this sentence:

If the correct block is already present locally and an alternative block for the same height is received, the alternative block must not be added to the currently active chain.